plugins/pipelines/admins/get.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const getSchema = joi.reach(schema.models.pipeline.base, 'admins').get;
+const idSchema = joi.reach(schema.models.pipeline.base, 'id');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/pipelines/{id}/admin',
+    config: {
+        description: 'Get the pipeline admin',
+        notes: 'Returns a pipeline admin record',
+        tags: ['api', 'pipelines'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', 'build', 'pipeline']
```

Comment:
shouldn't `temporal` token have access ? since that's what's used by the `queue-service` ?
Ideally we should have a token which represents `queue-service`

Reply:
Correct, adding temporal cope